Problem1.java

```diff
@@ -0,0 +1,51 @@
+import java.util.ArrayList;
+import java.util.List;
+
+// # Problem 1
+// Binary Tree Level Order Traversal (https://leetcode.com/problems/binary-tree-level-order-traversal/)
+
+
+ // Definition for a binary tree node.
+ public class Problem1 {
+      int val;
+      Problem1 left;
+      Problem1 right;
+     Problem1() {}
+      Problem1(int val) { this.val = val; }
+      Problem1(int val, Problem1 left, Problem1 right) {
+          this.val = val;
+          this.left = left;
+          this.right = right;
+     }
+ }
+
```

Comment:
Please follow the template given for competitive coding
Answer all these questions (you have not answered last 2 questions)
Time Complexity 
Space Complexity 
Did this code successfully run on Leetcode
Any problem you faced while coding this